.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+on: [pull_request]
+
+jobs:
+    buildinfo:
+        runs-on: ubuntu-latest
+        container: debian:bullseye
+        steps:
+            - name: Install dependencies
+              run: |
+                apt-get update && apt-get install python3-debian git git-lfs -y
+            - name: Checkout
+              uses: actions/checkout@v3
+              with:
+                lfs: true
+                fetch-depth: 0
+            - name: Check buildinfo
+              run: |
+                git config --global --add safe.directory '*'
+                git clone https://github.com/freedomofpress/securedrop-builder --depth 1
+                git clone https://github.com/freedomofpress/build-logs --depth 1
```

Comment:
How would you feel about adding a comment that explains the triangulation here?

1. `securedrop-builder` provides `check-buildinfo`, which checks...
2. `securedrop-apt-test` in `.` against...
3. `build-logs`

The reusability of (1) and the indirection of (3) make sense, but I had to think about it for a moment.

Reply:
Done, hopefully it's clearer now.